### DISCUSSION TITLE
Remix SPA Mode Sample

### DISCUSSION THREAD

**Discussion by YusukeSano** [refinedev/refine#5963] [Q&A] [answered]
Hello👋

I am currently working with the Refine and exploring the use of Remix in SPA mode. I am looking for a sample code or an example project that demonstrates how to set up and use Remix in SPA mode within the Refine.

I noticed that Issue #5202, #5734, which addresses support for SPA mode, has been closed. If there are any updates or resources related to this issue, please let me know.

If anyone has experience with this setup or can provide any sample code, it would be greatly appreciated.

Thank you!

**Comment by BatuhanW** [comment, marked as answer]
Hey @YusukeSano we don't have SPA example yet. Feel free to submit a PR if you put an example together, we'd be happy to see your contribution.
